### PR TITLE
MAT-6074 Re-write QI-Core test case Highlighting tab - Layout - Populations Only

### PR DESCRIPTION
**Pull request by adongare**
## MADiE PR

Jira Ticket: [MAT-6074](https://jira.cms.gov/browse/MAT-6074)
(Optional) Related Tickets: [MAT-6268](https://jira.cms.gov/browse/MAT-6268) QI-Core test case Highlighting tab - Layout - Multiple Population Criteria

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

https://github.com/MeasureAuthoringTool/madie-patient/assets/23436747/778c3d43-3a52-4114-a96e-6a7906c06c28

### CODE REVIEW

File: src/components/editTestCase/groupCoverage/GroupCoverage.tsx [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                opacity: 0.6,
+              },
+            },
+            "& .MuiSelect-icon": {
+              color: "#515151",
+              fontSize: "large",
+            },
+          }}
+          inputProps={{
+            "data-testid": "population-criterion-input",
```
Comment (sb-prateekkeerthi) [inline src/components/editTestCase/groupCoverage/GroupCoverage.tsx:186]: why are we adding all the properties of select again. aren't they already present in the design systems ?

**Review by sb-prateekkeerthi — CHANGES_REQUESTED** [review]

**Review by sb-cecilialiu — DISMISSED** [review]

File: src/components/editTestCase/groupCoverage/GroupCoverage.tsx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    const populationResults = getPopulationResults(criteriaId);
+    setPopulationResults(populationResults);
+    const group = groupPopulations.find((gp) => gp.groupId === criteriaId);
+    setSelectedPopulation(getFirstPopulation(group));
+  };
+
+  const changePopulation = (population: Population) => {
+    setSelectedPopulation(population);
+    const result = populationResults?.find(
+      // TODO: Handle 2 IP scenario
```
Comment (sb-prateekkeerthi) [inline src/components/editTestCase/groupCoverage/GroupCoverage.tsx:120]: do we have a separate story for handling 2 IPs ?

**Review by sb-prateekkeerthi — DISMISSED** [review]

File: src/components/editTestCase/groupCoverage/GroupCoverage.tsx (hunk 2 above)
Comment (adongare) [reply src/components/editTestCase/groupCoverage/GroupCoverage.tsx:120]: No, we'll have to create one

**Review by sb-prateekkeerthi — APPROVED** [review]

File: src/components/editTestCase/groupCoverage/GroupCoverage.tsx (hunk 1 above)
Comment (adongare) [reply src/components/editTestCase/groupCoverage/GroupCoverage.tsx:186]: this is a slightly different select box with no borders. It is kind of unsetting the default CSS.

**Pull request merged**